files/squid_whitelist/web_whitelist

```diff
@@ -49,6 +49,13 @@ docker-auth.elastic.co
 docker.elastic.co
 docs.google.com
 download.java.net
+qualysguard.qg3.apps.qualys.com:443
+distribution.qg3.apps.qualys.com:443
+monitoring.qg3.apps.qualys.com:443
+qgadmin.qg3.apps.qualys.com:443
+scanservice1.qg3.apps.qualys.com:443
+qualysapi.qg3.apps.qualys.com:443
+qagpublic.qg3.apps.qualys.com
```

Comment:
You don't need the port, the format is wrong. Also I _think_  you can remove these altogether since the `web_wildcard_whitelist` file should cover all domains under `.apps.qualys.com`

Reply:
I've changed the format to .qg3.apps.qualys.com under web_whitelist and web_wildcard_whitelist